stellar/deploy.testnet.sh

```
@@ -2,19 +2,20 @@ bash build.sh
 
 echo "lists contract id: $lists_contract_id"
 echo "registry contract id: $registry_contract_id"
 echo "round & factory contract id: $round_contract_id"
 echo "deployer address: $my_address"
 
-stellar contract invoke --source potlock-testnet --network testnet --id ${lists_contract_id} -- initialize --owner ${my_address} 
-stellar contract invoke --source potlock-testnet --network testnet --id ${registry_contract_id} -- initialize --contract_owner ${my_address} 
-stellar contract invoke --source potlock-testnet --network testnet --id ${round_contract_id} -- initialize --registry_address ${registry_contract_id} --caller ${my_address} --token_address ${native_token} --protocol_fee_basis_points 200 --protocol_fee_recipient ${my_address} --default_page_size 5
+stellar contract invoke --source potlock-testnet --network testnet --id ${lists_contract_id} -- initialize --owner ${my_address}
+stellar contract invoke --source potlock-testnet --network testnet --id ${lists_contract_id} -- create_list --owner ${my_address} --name "Test List" --default_registration_status Approved
+stellar contract invoke --source potlock-testnet --network testnet --id ${registry_contract_id} -- initialize --contract_owner ${my_address}
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)